docs/source/upcoming_release_notes/1054-radiation-sensor.rst

```diff
@@ -0,0 +1,30 @@
+1054 radiation-sensor
+#################
+
+API Changes
+-----------
+- Added radiation.py
+
+Features
+--------
+- Container for reading out 6030 radiation data
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- HPI6030
```

Comment:
Give https://github.com/pcdshub/pcdsdevices/blob/master/docs/source/upcoming_release_notes/template-full.rst a quick peek, I think all the stuff in this doc is good but it belongs in the "New Devices" section, for example:
```suggestion
- Added HPI6030 in radiation.py, a device for reading out 6030 radiation data.
```